docs/data/data-grid/list-view/list-view.md

```diff
@@ -16,14 +16,107 @@ Unlike the default grid view, the list view makes no assumptions on how data is
 
 In order to display data in a list view, a `unstable_listColumn` prop must be provided with a `renderCell` function.
 
-{{"demo": "ListView.js", "bg": "inline"}}
+```tsx
+function ListViewCell(params: GridRenderCellParams) {
+  return <>{params.row.id}</>;
+}
+
+const listColDef: GridListColDef = {
+  field: 'listColumn',
+  renderCell: ListViewCell,
+};
+```
+
+{{"demo": "ListView.js", "bg": true}}
+
+## Enable with a media query
+
+The `useMediaQuery` hook from `@mui/material` can be used to enable the list view feature at a specified breakpoint.
+
+```tsx
+import * as React from 'react';
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+function App() {
+  const theme = useTheme();
+  const isListView = useMediaQuery(theme.breakpoints.down('md'));
+
+  return <DataGrid unstable_listView={isListView} />;
+}
+```
+
+## Editable rows
+
+The [editing](/x/react-data-grid/editing/) feature is not supported in list view, however, it is possible to build an editing experience from within your custom cell renderer.
+
+{{"demo": "ListViewEdit.js", "bg": true}}
+
+## Field visibility
+
+Similar to the default grid view, field visibilty can be toggled in list view.
+
+In the list view column’s `renderCell` function, data can be conditionally rendered based on [column visibility](/x/react-data-grid/column-visibility/).
+
+```tsx
+import {
+  useGridSelector,
+  useGridApiContext,
+  gridColumnVisibilityModelSelector,
+} from '@mui/x-data-grid';
+
+function ListViewCell(params: GridRenderCellParams) {
+  const apiRef = useGridApiContext();
+  const columnVisibilityModel = useGridSelector(
+    apiRef,
+    gridColumnVisibilityModelSelector,
+  );
+  const showCreatedAt = columnVisibilityModel.createdBy !== false;
+
+  return (
+    <>
+      <span>{params.row.id}</span>
+      {showCreatedAt && (
+        <time datetime={params.row.createdAt}>
+          {formatDate(params.row.createdAt)}
+        </time>
+      )}
+    </>
+  );
+}
+```
 
 ## Advanced usage
 
 The list view feature can be combined with [custom subcomponents](/x/react-data-grid/components/) to provide an improved user experience on small screens.
 
-{{"demo": "ListViewAdvanced.js", "bg": "inline", "iframe": true, "maxWidth": 360, "height": 600, "hideToolbar": true}}
+{{"demo": "ListViewAdvanced.js", "iframe": true, "maxWidth": 360, "height": 600}}
+
+## Feature support
```

Comment:
How do we feel about this section?

Reply:
I'm cool with it. Maybe "Feature interoperability" would be more accurate since we're specifically describing how this feature works with other features. But that's also a pretty gnarly word to throw at folks 😅 especially non-native English speakers.

Reply:
That is gnarly 🤙.

What do others think? @mui/xgrid do we have a similar section anywhere else?

Reply:
For me, `Feature compatibility` would be more understandable.

Another possibility could be to convert the section to a `::success` callout following the pattern for `::warning` callout in https://mui.com/x/react-data-grid/column-spanning/, I'd prefer keeping it as a separate section though.

Reply:
I like "Feature compatability", makes more sense than "support" when comparing to other features 👍

Reply:
I like "Feature compatibility", makes more sense than "support" when comparing to other features 👍